### PR TITLE
Updated documentation for version 5.0.0.

### PR DESCRIPTION
**Pull request by henryso**
Please review and merge if satisfactory.

### CODE REVIEW

File: doc/Command_Index_internal.tex [hunk 1]
```diff
@@ -896,6 +892,44 @@ \section{Gregorio\TeX{} Controls}
...
+used to initialize \verb=\gre@bolshiftcleftypelocal= at the start of a score.
+
+\macroname{\textbackslash gre@bolshiftcleftypelocal}{}{gregoriotex-spaces.tex}
+Takes on the value \texttt{1}, \texttt{2}, or \texttt{3} depending on whether
+to calculate the beginning-of-line shift based on the largest, current, or
+forced clef width (\ie, directly use the value stored in
+\verb=\gre@dimen@clefwidth@bol=), respectively.  This is the ``local'' value,
+used in the actual computation.
+
+\macroname{\textbackslash gre@update@clefwidth@current}{\#1}{gregoriotex-spaces.tex}
```
Comment (rpspringuel) [inline doc/Command_Index_internal.tex]: Macro name is wrong.  Should be `largest`, not `current`.

**Comment by henryso** [comment]
Done.  Let me know if you find anything else.

File: doc/Command_Index_internal.tex [hunk 2]
```diff
@@ -896,6 +892,44 @@ \section{Gregorio\TeX{} Controls}
...
+Updates the ``current'' clef width, used when \verb=\gre@bolshiftcleftypelocal=
+is \texttt{2}.
+
+\begin{argtable}
+  \#1 & distance & The new value to use for the ``current'' clef width.\\
+\end{argtable}
+
+\macroname{\textbackslash gre@update@clefwidth@forced}{\#1}{gregoriotex-spaces.tex}
+Updates the ``largest'' clef width, used when \verb=\gre@bolshiftcleftypelocal=
```
Comment (rpspringuel) [inline doc/Command_Index_internal.tex]: Is this description right?  Should `largest` be `forced`?

**Comment by henryso** [comment]
Thanks again.  Anything else?

**Comment by rpspringuel** [comment]
Everything looks good to me now.  Thanks for doing this.

**Pull request merged**